nextcord/iterators.py

```diff
@@ -521,7 +521,7 @@ def __init__(
 
         self.reverse: bool
         if oldest_first is None:
-            self.reverse = after is not None
+            self.reverse = False
         else:
             self.reverse = oldest_first
```

Comment:
Can't you just set `reverse` to `bool(oldest_first)` instead? `None` would be evaluated to `False`.
I would make a suggestion for this, but GitHub doesn't allow me to make suggestions with deleted lines.